src/@apollo/client/cache/inmemory/ApolloClient__Cache_InMemory_Policies_FieldPolicy.res

```diff
@@ -4,13 +4,6 @@ module FieldNode = ApolloClient__Graphql.Language.Ast.FieldNode
 module ReadFieldFunction = ApolloClient__Cache_Core_Types_Common.ReadFieldFunction
 module ToReferenceFunction = ApolloClient__Cache_Core_Types_Common.ToReferenceFunction
 
-module FieldMergeFunction = {
-  module Js_ = {
-    type t
-  }
-  type t = Js_.t
-}
-
```

Comment:
I had to move this type further down in the module to share the `FieldFunctionOptions.t` type.